Tests/Listener/ORM/MediaEventSubscriberTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\Listener\ORM;
+
+use Doctrine\ORM\Events;
+use Sonata\MediaBundle\Listener\ORM\MediaEventSubscriber;
+use Sonata\MediaBundle\Model\Media;
+use Sonata\MediaBundle\Provider\Pool;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @author Mathieu Lemoine <mlemoine@mlemoine.name>
+ */
+class MediaEventSubscriberTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+    * sonata-project/SonataClassificationBundle#60
+    * sonata-project/SonataMediaBundle#780.
```

Comment:
ping @sonata-project/contributors do we keep this?

or do we add `@see` with a URL to the issue?